JSTests/stress/arguments-define-property-throws-out-of-memory.js

```diff
@@ -19,6 +19,7 @@ function foo() {
 
 let s = [0.1].toLocaleString().padEnd(2 ** 31 - 1, 'ab');
 
-shouldThrow(() => {
+// Do not crash
+try {
     foo(s);
-}, `RangeError: Out of memory`);
+} catch { }
```

Comment:
Could you please describe how this change is relevant to the patch? Thanks.

Reply:
Yup, done.